packages/matter.js/src/device/PairedNode.ts

```diff
@@ -374,7 +374,12 @@ export class PairedNode {
             updateTimeoutHandler: async () => {
                 logger.info(`Node ${this.nodeId}: Subscription update not received ...`);
                 try {
-                    await this.subscribeAllAttributesAndEvents();
+                    const { attributeChangedCallback, eventTriggeredCallback } = this.options;
```

Comment:
This.options or better using what comes in in this method as params?

Reply:
yeah, i totally spaced this was actually calling itself and already had the options  passed in.